assopy/models.py

```diff
@@ -458,37 +458,30 @@ def _applyToTotal(self, total, guard):
         return -1 * discount
 
 
-class OrderManager(models.Manager):
-    def get_queryset(self):
-        return self._QuerySet(self.model)
+class OrderQuerySet(models.QuerySet):
+    def use_coupons(self, *coupons):
+        return self.filter(orderitem__ticket=None, orderitem__code__in=(c.code for c in coupons))
 
-    def __getattr__(self, name):
-        return getattr(self.all(), name)
+    def conference(self, conference):
+        return self.filter(orderitem__ticket__fare__conference=conference).distinct()
 
-    class _QuerySet(QuerySet):
-        def use_coupons(self, *coupons):
-            return self.filter(orderitem__ticket=None, orderitem__code__in=(c.code for c in coupons))
-
-        def conference(self, conference):
-            return self.filter(orderitem__ticket__fare__conference=conference).distinct()
-
-        def usable(self, include_admin=False):
-            """
-            restituisce tutti gli ordini "usabili", cioè tutti gli ordini con
-            metodo bonifico (a prescindere se risultano pagati o meno) e tutti
-            gli ordini con metodo paypal (o cc) completati.
-            """
-            qs = self.filter(models.Q(method='bank')|models.Q(method__in=('cc', 'paypal'), _complete=True))
-            if include_admin:
-                qs = qs.filter(method='admin')
-            return qs
+    def usable(self, include_admin=False):
+        """
+        restituisce tutti gli ordini "usabili", cioè tutti gli ordini con
+        metodo bonifico (a prescindere se risultano pagati o meno) e tutti
+        gli ordini con metodo paypal (o cc) completati.
+        """
+        qs = self.filter(Q(method='bank') | Q(method__in=('cc', 'paypal'), _complete=True))
```

Comment:
doesn't it basically mean `.filter(method__in=['bank', 'cc', 'paypal', _complete=True)`?

Reply:
Actually, taking it a level of abstraction higher, as the method is buggy and the implementation does not match the implementation (_Returns all "usable" orders, ie all orders with bank transfer method  of whether they are paid or not) and all  with paypal (or cc) method completed._) - this queryset method is not used anyhwere. I'll just remove it.

Reply:
LGTM